src/prefect/engine/state.py

```diff
@@ -49,9 +49,9 @@ def __init__(self, message: str = None, result: Any = NoResult):
 
     def __repr__(self) -> str:
         if self.message:
-            return '{}("{}")'.format(type(self).__name__, self.message)
+            return '<{}: "{}">'.format(type(self).__name__, self.message)
```

Comment:
Actually I think we can use this `repr` regardless of `self.message`.

Reply:
Correct me if I'm wrong, but wouldn't that seem weird in case there's no message, or in the case of the `State` class itself which has no such attribute?

Reply:
Good question - actually all `State` classes (including the base) have a `message` attribute.  More often than not, the states have messages associated with them, but when they don't, the repr will actually look like:
```python
'<Success: None>'
```
which I think is reasonable.

Reply:
Oh okay, that's great. I'll change this right away.

Reply:
No rush! FYI I will probably be away from my computer for the rest of the day so apologies if I don't respond as quickly to future updates.

Reply:
No problem!